src/pages/api/breaches.ts

```diff
@@ -0,0 +1,53 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import axios from 'axios';
+import { HaveIbeenPwnedText } from '@/assets/types/have-i-been-pawned';
+const CACHE_CLEAN_INTERVAL_MS = 2 * 60 * 60 * 1000;
+interface BreachesProps {
+  textContent: HaveIbeenPwnedText['HeroSection']['breaches'];
+}
+
+const API_URL = process.env.HIBP_API_URL;
```

Comment:
Use simple names for the env variables. It should be easy to understand what it contains.

Reply:
Done